src/main/java/com/mulberry/blackmulberry/controller/main/AnyController.java

```diff
@@ -7,7 +7,7 @@
 public class AnyController {
```

Comment:
i believe that its not good name for controller. please change it. you can rename for excample Main or Home.